nl.json

```diff
@@ -139,54 +148,126 @@
             "more-ingame-button" : "Meer Laden"
         },
         "randomiser" : {
-            "title" : "Willekeuriger",
+           "title" : "Willekeurige items",
```

Comment:
The spacing was correct before. Please don't remove the space before the translation key.

Reply:
Oops, probably my fault! Of course there has to be a space before the translation key.